packages/docs/src/examples/v-snackbar/prop-variants.vue

```diff
@@ -23,18 +23,31 @@
       rounded="pill"
       top
     >
-      Lorem ipsum dolor sit amet consectetur.
+      snackbar with <strong>rounded: pill</strong> property.
```

Comment:
This feels very repetitive, maybe just `rounded="pill"` (in a code block?)

```suggestion
      rounded="pill"
```

```suggestion
      `rounded="pill"`
```

```suggestion
      `<v-snackbar rounded="pill">`
```